build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+    ext.kotlin_version = '1.4.10'
```

Comment:
1.4.20 is the latest.
Also, we should upgrade the `com.android.tools.build:gradle` plugin to the latest stable.

Reply:
Updated kotlin in https://github.com/eduvpn/android/pull/315/commits/28e33e26738885b6f23114b6140602369ce05ea2